app/.gitignore

```diff
@@ -1,3 +1,4 @@
 /build
 
-*.propeties
\ No newline at end of file
+*.propeties
+/src/main/assets/koin.properties
```

Comment:
@aniketbhoite remove koin.properties as koin is no longer part of this project.

Reply:
Done